tests/lib/test_appliance.py

```diff
@@ -109,7 +104,7 @@ def display(results, verbose):
         for filename in filenames:
             sys.stdout.write('-'*75+'\n')
             sys.stdout.write('%s:\n' % filename)
-            data = open(filename, 'rb').read()
+            data = open(filename, 'r', errors='replace').read()
```

Comment:
I think the kwar `errors=` here is the source of the Travis failusres. You could use `io.open()` instead for Python 2 compatibility.  I believe it is available on all supported Python versions.

https://docs.python.org/3/library/io.html#io.open